contracts/eos/BancorConverter/migrate.cpp

```diff
@@ -0,0 +1,79 @@
+[[eosio::action]]
+void BancorConverter::migrate( const set<symbol_code> converters )
+{
+    for ( const symbol_code symcode : converters ) {
```

Comment:
do we need to add any authorization check here or can anyone call it?

Reply:
I was going to add a `get_self()` authorization check for this, however this `migrate` doesn't do much other then just copying over data, so technically anyone can do this and it won't break anything.

But we can add `require_auth(get_self())` to make sure it's only the contract that is authorized to push `migrate`